packages/button/styles/variant-ghost.css

```diff
@@ -0,0 +1,63 @@
+.g-btn.variant-ghost {
+  color: var(--gray-3);
```

Comment:
Would have loved to just set a custom property here but given the rest of the SVG dependencies below, it felt like a little too fancy to introduce yet another styling pattern for this component

Reply:
Could the SVG dependencies below make use of `currentColor`? I could see that be the default in the index.css file and allow override if needed within each variant. Probably a bigger change than what you might want to do here, but it could reduce some of the duplication across the variants.

Reply:
Yeah that's a solid idea!

I would consider it a different effort since this component could also take advantage of css-modules and potentially some other upgrades / debt removal. 

css-modules alone feels like it might compound the amount of debt cleanup we'd need to do across sites as well.

Reply:
Just a +1 to all the above - between `currentColor` and potential use of custom properties since we dropped IE11 support, a refactor here could probably cut down on duplicated CSS quite a bit, and probably best as a separate PR & effort since it'll be a bigger change 🌊